lib/railway_ui_web/endpoint.ex

```diff
@@ -5,6 +5,8 @@ defmodule RailwayUiWeb.Endpoint do
     websocket: true,
     longpoll: false
 
+  socket "/live", Phoenix.LiveView.Socket
```

Comment:
is this just needed for Phoenix upgrade? I'm 100% very hesitant to add anymore LiveView code to any of our apps (outside of Course Conductor). I think the tech debt of adding it at this point in it's lifecycle is not wise for our teams. Very heavy push back from me to support more apps using it with where it is at and what problems are team needs to solve.

Reply:
For this kind of stuff (admin views at least) I'm personally OK with live view. 
@Lukeghenco what kind of issues are you concerned about?

Reply:
I'm concerned about maintaining code on some so new and has such a small community around it. I don't want a hybrid app with JS, HTML and LiveView existing for a team of our size to maintain. Let's pick the tool that we know how to use and we can build on quickly. This feels like a shiny new object that is cool, but not a smart move for us yet. 

The argument for admin views is a case, but it's still something that 1. Much harder to test than what we get with React, 2. Slows us down if the use case doesn't fit with LiveView (and or lib/tool exist to use with it). 3. a ton more complexity for the team to learn and grok when reading.

Reply:
Maybe it's because I've been playing with it lately, but the learning curve isn't as high as coming new to react and co. Live views are essentially GenServers that maintain an update state. The api is small, and pretty easy to grok. I think the LV code we have was built when the lib was really unstable. It didn't even have JS hooks. All that to say, take a look at it today.

Reply:
I think this is worth a bigger team discussion outside of a Github PR review. I'm 100% dug in the sand at the moment on not adding more LiveView at the moment for many reasons, but willing to give a few hours into using it to see if things have changed and it actually solves more problems than it brings, but also has the correct ways to test DOM interaction that isn't all implementation detail testing. 

But from what I played with even as of a month ago, I was not feeling it outside of the core examples that let's try everything in Elixir, Elixir devs were trying to show it off with. 

But let's chat IRL over this more. I have many thoughts and concerns. I ultimately think it is an unwise decision for our team right now and brings more work and learning then we need. Down to learn new things, but not quite down to add that as tech debt.